source/patterns/@aws-solutions-constructs/aws-lambda-kendra/README.md

```diff
@@ -0,0 +1,133 @@
+# aws-lambda-kendra module
+
+* * *
+
+![Stability: Experimental](https://img.shields.io/badge/stability-Experimental-important.svg?style=for-the-badge)
+
+> All classes are under active development and subject to non-backward compatible changes or removal in any
+> future version. These are not subject to the [Semantic Versioning](https://semver.org/) model.
+> This means that while you may use them, you may need to update your source code when upgrading to a newer version of this package.
+
+* * *
+
+| **Reference Documentation**: | <span style="font-weight: normal;">https://docs.aws.amazon.com/solutions/latest/constructs/</span> |
+| --- | --- |
+
+| **Language** | **Package** |
+| --- | --- |
+| ![Python Logo](https://docs.aws.amazon.com/cdk/api/latest/img/python32.png) Python | `aws_solutions_constructs.aws_lambda_kendra` |
+| ![Typescript Logo](https://docs.aws.amazon.com/cdk/api/latest/img/typescript32.png) Typescript | `@aws-solutions-constructs/aws-lambda-kendra` |
+| ![Java Logo](https://docs.aws.amazon.com/cdk/api/latest/img/java32.png) Java | `software.amazon.awsconstructs.services.lambdakendra` |
+
+## Overview
+
+This AWS Solutions Construct implements an AWS Lambda function and Amazon Kendra index with the least privileged permissions.
+
+Here is a minimal deployable pattern definition:
+
+Typescript
+
+```typescript
+import { Construct } from 'constructs';
+import { Stack, StackProps, Aws } from 'aws-cdk-lib';
+import { LambdaToKendra } from '@aws-solutions-constructs/aws-lambda-kendra';
+import * as lambda from "aws-cdk-lib/aws-lambda";
+import * as s3 from "aws-cdk-lib/aws-s3";
+
+const lambdaProps: lambda.FunctionProps = {
+  code: lambda.Code.fromAsset(`lambda`),
+  runtime: lambda.Runtime.NODEJS_18_X,
+  handler: 'index.handler'
+};
+
+new LambdaToKendra(this, 'sample', {
+  lambdaFunctionProps: lambdaProps,
+  kendraIndexProps: {},
+  kendraDataSourceProps: [{
+    type: 'S3',
+    dataSourceConfiguration: {
+      s3Configuration: {
+        bucketName: 'your-bucket-name',
+      }
+    }
+  ],
+});
+```
+
+Python
+
+```python
+TBD
+```
+
+Java
+
+```java
+TBD
+```
+
+## Pattern Construct Props
+
+| **Name** | **Type** | **Description** |
+| --- | --- | --- |
+| existingLambdaObj? | [`lambda.Function`](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_lambda.Function.html) | Existing instance of Lambda Function object, providing both this and `lambdaFunctionProps` will cause an error. |
+| lambdaFunctionProps? | [`lambda.FunctionProps`](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_lambda.FunctionProps.html) | User provided props to override the default props for the Lambda function. |
+| kendraIndexProps? | [`kendra.CfnIndexProps`](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_kendra.CfnIndex.html) | Optional user provided props to override the default props for the Kendra index. **Is this required?** |
```

Comment:
Questioning the `**Is this required?**` comment at the end of of the `kendraIndexProps`?